Packs/Core/Playbooks/playbook-Uncommon_creation_or_access_operation_of_sensitive_shadow_copy_by_a_high_risk_process.yml

```diff
@@ -0,0 +1,971 @@
+          "y": 845
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "31":
+    id: "31"
+    taskid: a5c00bd3-eda3-4a80-8db7-160828d6bc8b
+    type: title
+    task:
+      id: a5c00bd3-eda3-4a80-8db7-160828d6bc8b
+      version: -1
+      name: Triage
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "6"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 705,
+          "y": -240
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "32":
+    id: "32"
+    taskid: 84fdb0e0-163f-4ecf-8545-b41727f8c970
+    type: regular
+    task:
+      id: 84fdb0e0-163f-4ecf-8545-b41727f8c970
+      version: -1
+      name: Get Causality process prevalence
+      description: Get the prevalence of a process, identified by process_name.
+      script: '|||core-get-process-analytics-prevalence'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "28"
+    scriptarguments:
+      process_name:
+        simple: ${alert.cgoname}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1330,
+          "y": 210
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "33":
+    id: "33"
+    taskid: db0e4684-aeb5-4d83-8279-e9ab7dcd5da7
+    type: title
+    task:
+      id: db0e4684-aeb5-4d83-8279-e9ab7dcd5da7
+      version: -1
+      name: Investigation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "7"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 110,
+          "y": 70
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "10_11_yes": 0.38,
+      "10_12_#default#": 0.4,
+      "26_27_#error#": 0.57,
+      "30_12_#default#": 0.33,
+      "7_10_cmd.exe|rundll32.exe": 0.42,
+      "7_13_#default#": 0.51,
+      "7_9_powershell.exe": 0.65,
+      "9_11_(powershell.exe | ntdsutil.exe | esentutl.exe | cmd.exe": 0.34
+    },
+    "paper": {
+      "dimensions": {
+        "height": 2200,
+        "width": 1920,
+        "x": -210,
+        "y": -385
+      }
+    }
+  }
+inputs: []
+outputs: []
+tests:
+- No tests (auto formatted)
+fromversion: 6.10.0
```

Comment:
teminate causality is not supported in those versions, please make it from version <the version that support terminate causality>

Reply:
same, what about that?